pkg/util/retry/backoff.go

```diff
@@ -53,3 +53,21 @@ func (b BackoffFunc) Reset() {}
 func (b BackoffFunc) Clone() Backoff {
 	return b
 }
+
+func DefaultExponentialBackoff() Backoff {
+	const (
+		waitMin        = 15 * time.Second // nolint: revive
+		waitMax        = 1 * time.Minute
+		maxElapsedTime = 15 * time.Minute
```

Comment:
Should we perhaps add some way for user to configure the time we wait for nodes to go below load threshold?
This could be useful, since some clusters might have a pattern of spikes that usually last for let's say about an hour.

Moreover, AFAIK Manager's scheduler also reschedules tasks that returned an error with exponential backoff, so should we take that into consideration when setting default value here?.